### ISSUE TITLE
[IDE] Scroll speed per mouse wheel is too slow.

### ISSUE THREAD

**Issue by y0nd0** [defold/defold#5169]
**Description**
The scroll speed in the IDE is ridiculously slow. I have to spin my mouse wheel like a madman.

**Reproduce**
1. Open a project.
2. Open the `readme.md` file or settings. (everywhere)
3. Scroll with your mouse wheel.

You'll see that the scroll speed is too slow. Nearly one pixel per wheel tick.

**Expected behavior**
It should be behave like default scrolling in Windows.
Or just increase the scroll value per wheel tick.

**Defold version:**
Version 1.2.172 - Editor 2.0

**Platforms:**
 - Platforms: IDE (Defold Editor 2.0)
 - OS: Windows 10 Pro 64bit
 - Device: Resolution 4k (ui scale 2x). But also tested in 1080p.

**Screenshots:**
![demo](https://user-images.githubusercontent.com/68586847/92319325-01015500-f017-11ea-968c-57a20c1f81e5.gif)

**Additional context:**
My finger hurts now. 😢 And my mouse wheel is broken.
Seriously, that's a problem.

**Comment by britzl** [comment]
Interesting. I don't believe this is a general problem for all users though as this is the first time I hear of this. Mouse driver or mouse settings?

**Comment by britzl** [comment]
Here's a mention of slow scroll speed fr JavaFX though: https://github.com/javafxports/openjdk-jfx/issues/417

@vlaaad do you have any input?

**Comment by y0nd0** [comment]
No special settings or drivers. I'm using the Windows 10 default settings and haven't changed any settings in the Defold or Java environment. Defold is the only programm with this issues. I don't know if this issue is related to the JFX issue. It's about "when content has similar size to container". I don't believe that this is the problem here. idk.

Test:
I changed the Windows settings for the mouse. "Select how many lines should be scrolled." (Not sure if this is the 1:1 text in english. It's not my OS language. Anyway, there is a setting for the scroll speed in Windows. Speed in lines.)
The default is "3", which works great (except in Defold). Increasing this to "100" (max value) effects to Defold. The scoll speed is better now. A good value is something between 30 - 50 lines. But with this setting value, the scrolling in other programs and OS is not acceptable. Too fast. ...

Result: 
There is a relation to the OS scroll speed setting. But the real issue is the program (Defold / JFX / ?)  itself. Maybe there is a wrong scale or line height. I see that the default OS setting only scrolls a line per wheel tick (more or less). 
I don't know what exactly the problem is. Maybe the fastest solution is to implement a scroll speed setting Defold. The best solution is to find a way to scroll with the same behavior as other programs in the OS. Or just set a scroll scale to 10x if possible. But not sure if sure if this covers all users. Maybe better scroll acceleration would be a good solution? Currently I can scoll as fast as I want, but the speed is linear. ...
I don't think that this has to do with hidpi ui scale of the OS. Beacuase I tested it in 4k (x2) and 1080p (x1). 

![demo](https://user-images.githubusercontent.com/68586847/92333338-e49c0180-f084-11ea-823b-5ce59389b8a3.gif)

**Comment by y0nd0** [comment]
Ok, I see the same behavior in the Task-Manager of Windows. (Btw. Defold remains in as Process after closing. But this is another issue.) Anyway, that's weird. Some scroll contains behave different (one which does not respect the scroll lines.). But generally I have no scroll issues. In Chrome, Windows Explorer, other IDEs, etc. All fine. Whatever, I close this because I see that scroll issue on the task manager too. I'll test it at another PC soon.

![demo](https://user-images.githubusercontent.com/68586847/92334142-95a59a80-f08b-11ea-9533-157650aa3148.gif)

**Comment by britzl** [comment]
Ok, thanks for the thorough investigation! Let us know how it works on another PC!

(and Defold should not stay as a lingering process after it is shut-down)

**Comment by y0nd0** [comment]
I have now tested it on another PC running Windows 10. The behavior is similar, but twice faster. You see the gif record above, it's a half line per wheel tick. On the other pc it's one line per tick. The scroll speed in Defold is slow in both cases. But it seems that this depends on the same scroll container as the task-manager. It's weird that this is a "normal" behavior. I would say this is a bug. A bad UX. Who reviewed that at Microsoft? Anyway, the scroll speed in the code editor in Defold is ok. Because this editor is based on lines. Other containers (like readme.md or settings) are based on content. But the Chrome browser is also based on content and scrolls perfect. Maybe there is a solution for Defold / JFX? (Maybe improve the acceleration?) I mean, this is the first time I've encountered such a problem in a program. It's not impossible to work with this scroll issue. But it's a very bad UX. Personally, I was just looking for a game engine and I'm not sure I'm using Defold. But it looks professional and clean. I'm considering doing something serious with it.
